custom_components/tahoma/entity.py

```diff
@@ -98,6 +98,7 @@ def device_info(self) -> DeviceInfo:
             sw_version=self.executor.select_attribute(CORE_FIRMWARE_REVISION),
             suggested_area=self.coordinator.areas[self.device.place_oid],
             via_device=self.executor.get_gateway_id(),
+            configuration_url=self.coordinator.client.server.configuration_url,
```

Comment:
Apparently, there is no effect on my env. I cannot see any url in my device pages.

Reply:
Are you sure you have the newest update + front-end reloaded?

Reply:
<img width="347" alt="Screen Shot 2021-11-03 at 21 51 04" src="https://user-images.githubusercontent.com/1424596/140190758-87735f04-fd93-46c9-b3bb-70d9eec12fd3.png">

Reply:
Ah yes,  I was looking for the wrong thing.